### DISCUSSION TITLE
Any way for users to see upcoming changes?

### DISCUSSION THREAD

**Discussion by K0-RR** [ConsistencyPlus/ConsistencyPlus#22] [Q&A] [answered]
I saw the https://github.com/ConsistencyPlus/ConsistencyPlus/blob/master/TO-DO.txt but there is no progress information there and it hasn't been updated for a while. Any expected release date for 0.3.0?

**Comment by Siuolplex** [comment, marked as answer]
Alright, so, most of our updates and stuff is talked about on our discord, which is in the readme. 
TO-DO.txt is probably gonna go byebye, most devs dont use it.
Also, we have an internal/private roadmap, but we will see if we make it public

**Comment by Cart3r1234** [comment]
We want to try and have 0.3 out sometime this month (maybe the 22nd since every update we've released has been on the 22nd of the month by coincidence lol) but nothing is concrete yet.  Like Siuol said, we'll look into making the roadmap public, as that would give people a better idea of what to expect for each update, and how much is left before a release.
